README.md

```diff
@@ -61,6 +61,7 @@ Prerequisites
     #for unit tests
     $luarocks install lunit
     $luarocks install luaposix
+    $luarocks install luasocket
 
```

Comment:
Thanks for your PR! Besides `luasocket `, let's also add `lua-cjson` that is also required to run the tests.

See: https://github.com/cloudflare/raven-lua/blob/master/.travis.yml#L31